src/Microsoft.AspNet.Razor.Runtime/TagHelperContentWrapperTextWriter.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+
+namespace Microsoft.AspNet.Razor.Runtime
+{
+    /// <summary>
+    /// Implements a <see cref="TextWriter"/> that wraps around <see cref="TagHelperContent"/>.
+    /// </summary>
+    public class TagHelperContentWrapperTextWriter : TextWriter
+    {
+        private Encoding _encoding;
+
+        /// <summary>
+        /// Instantiates a new instance of <see cref="TagHelperContentWrapperTextWriter"/>.
+        /// </summary>
+        /// <param name="encoding">The <see cref="TextWriter"/>'s encoding.</param>
+        public TagHelperContentWrapperTextWriter(Encoding encoding)
+        {
+            _encoding = encoding;
```

Comment:
No.. The `Encoding` has only the getter. This is how it seems to be done in other TextWriter implementations also. - https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/Rendering/StringCollectionTextWriter.cs#L35